### PR TITLE
Discussion06

### PR DESCRIPTION
**Pull request by Exorust**

### CODE REVIEW

File: discussions/discussion06/discussion6.ipynb [hunk 1]
```diff
@@ -0,0 +1,440 @@
+{
```
Comment (sladenheim) [inline discussions/discussion06/discussion6.ipynb:1]: <p>Latex isn't rendering properly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/ds701_fa2024_assignments/pull/2/discussion/#comment-1793464027'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (sladenheim) [inline discussions/discussion06/discussion6.ipynb:1]: <p>Need to fix the latex rendering.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/ds701_fa2024_assignments/pull/2/discussion/#comment-1793464030'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (Exorust) [reply discussions/discussion06/discussion6.ipynb:1]: <p>Made it work</p>

Comment (Exorust) [reply discussions/discussion06/discussion6.ipynb:1]: <p>Made it work</p>

**Pull request merged**